float64 r11
float64 r12
float64 r13
float64 r21
float64 r22
float64 r23
float64 r31
float64 r32
float64 r33
float64 x
float64 y
float64 z
---
float64 theta1
float64 theta2
float64 d3
